drivers/serial/uart_nrfx_uart.c

```diff
@@ -24,17 +24,15 @@ static uart_irq_callback_t m_irq_callback; /**< Callback function pointer */
  *
  * @param dev UART device struct
  * @param baudrate Baud rate
- * @param sys_clk_freq_hz System clock frequency in Hz
```

Comment:
This change (as well as a few more below) should not be contained in this commit as it has nothing to do with the nrf5 renaming. Move all the changes related to shim refactoring to the first commit.